### PR TITLE
BaSP-TG-21: React router implementation

### PR DESCRIPTION
**Pull request by LucianaPrat**
**React Router was implemented to navigate through the different links of the web page, for the Nav and the Footer.
Fixed styles in Timesheet entity.**

**To implement react Routers in each entity:**
-	Import { useHistory } from ‘react-router-dom’
-	A variable is declared let history = useHistory()
-	it would be changed window.location.assign(‘/path’) by  history.push(‘/path’)
-	In case an ID is needed it would be history.push(‘/path’, { id: el ID que se necesite });
-	To use the ID -> history.location.state?.id

In TimeSheet it is already implemented to use it as an example

### CODE REVIEW

**Review by Baldizan — APPROVED** [review]

File: src/Components/Employees/index.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ const Employees = () => {
               <td>{employee.lastName}</td>
               <td>{employee.phone}</td>
               <td>{employee.email}</td>
-              <td>{employee.project.name}</td>
+              <td>{employee.project?.name}</td>
```
Comment (martumoser) [inline src/Components/Employees/index.js:64]: why do you change this on this pr?

File: src/Components/Footer/index.js [hunk 2]
```diff
@@ -9,22 +10,22 @@ function Footer() {
         </div>
         <ul className={styles.rutes}>
           <li>
-            <a href="/admins">admins</a>
+            <Link to="/admins">admins</Link>
```
Comment (martumoser) [inline src/Components/Footer/index.js:13]: Please add uppercase to the first letter: Admins, Super admins, etc

File: src/Components/Header/index.js [hunk 3]
```diff
@@ -32,22 +33,25 @@ function Header() {
         </div>
         <ul className={styles.rutes}>
           <li>
-            <a href="/admins">admins</a>
+            <Link to="/">home</Link>
```
Comment (martumoser) [inline src/Components/Header/index.js:36]: Please use uppercase on the first letter: Home, Admins, etc

**Review by martumoser — CHANGES_REQUESTED** [review]

File: src/Components/Employees/index.js (hunk 1 above)
Comment (LucianaPrat) [reply src/Components/Employees/index.js:64]: It showed me an error, and it was that when modifying the database there were projects without a name, I could revert it without problem

**Review by martumoser — APPROVED** [review]

**Review by mr-valen — APPROVED** [review]

**Review by Martinzancocchia — APPROVED** [review]

**Pull request merged**